runtime/crab/src/pallets/election_provider_multi_phase.rs

```diff
@@ -47,7 +47,7 @@ impl Config for Runtime {
 	type EstimateCallFee = TransactionPayment;
 	type Event = Event;
 	type Fallback = NoFallback<Self>;
-	type ForceOrigin = EnsureRootOrHalfCouncil;
+	type ForceOrigin = AtLeastHalf<CouncilCollective>;
```

Comment:
Root is different with `HalfCouncil` IMO.

And removing sudo does not indicate we do not have `ROOT` which could be from democracy

Reply:
Democracy's origin is root.

Reply:
IMO democracy can control everything.

Reply:
> IMO democracy can control everything.

I have different conclusions, ROOT can not do everything if the dispatch call requires other origin such as ensure_signed ensure_none etc.

Reply:
Democracy can do `system::set_storage`, which almost equals everything.

Reply:
And I think it is reasonable to fire up a referendum to kick out a tech.comm member.

Reply:
> Democracy can do `system::set_storage`, which almost equals everything. So I think it is meaningless to limit the democracy's power.

I know it is technically feasible, but I still don't like to use `system::set_storage`, because its bad readability and not easy for governance members to proposal and review.

Reply:
Did you check this?
It contains root already.
```
pub type Root = EnsureRoot<AccountId>;

pub type AtLeastHalf<Collective> =
	EnsureOneOf<AccountId, Root, EnsureProportionAtLeast<_1, _2, AccountId, Collective>>;
```

Reply:
> 

Ah I see, but please reflect it in the naming.